### ISSUE TITLE
website docs for Copy Tasks for Output Targets are incorrect

### ISSUE THREAD

**Issue by evaneus** [ionic-team/stencil#2654]
**Stencil version:**

```
 @stencil/core@1.17.3
```

**I'm submitting a:**

[ x ] bug report

**Current behavior:**

The [docs](https://stenciljs.com/docs/copy-tasks) for copy tasks says:
```Each output target can have its own copy config, which is an array of objects that defines any files or folders that should be copied over to the output target's build directory.```

But `docs` target does not support `copy`.

**Expected behavior:**

Docs to not say that each output target can have a copy config.
Or docs to support copy task.

**Steps to reproduce:**

Type error if you add `copy` to output target with type `docs` in stencil.config.ts

**Comment by rwaskiewicz** [comment]
Good catch! I apologize for the time it took for someone on the team to get back to you on this.

Although `docs` is no longer a valid output target in newer versions of Stencil, we can certainly clarify that the `docs-*` output targets do not have this task today.
